pkg/innerring/processors/netmap/process_peers.go

```diff
@@ -47,6 +51,28 @@ func (np *Processor) processAddPeer(ev netmapEvent.AddPeer) {
 		return
 	}
 
+	var info clientcore.NodeInfo
+	err = clientcore.NodeInfoFromRawNetmapElement(&info, netmapCore.Node(nodeInfo))
+	if err != nil {
+		np.log.Warn("could not convert node info: %w", zap.Error(err))
+		return
+	}
+
+	c, err := np.clientSource.Get(info)
+	if err != nil {
+		np.log.Warn("could not create client to the node: %w",
+			zap.Error(err),
+			zap.String("node", hex.EncodeToString(info.PublicKey())))
+		return
+	}
+
+	_, err = c.EndpointInfo(context.Background(), client.PrmEndpointInfo{})
```

Comment:
@roman-khimov, @cthulhu-rider, @AliceInHunterland, ping me if you think we need to check returned values (IMO, that is useless)

Reply:
I'd check them against the data for the map.

Reply:
added some checks